mathics/builtin/algebra.py

```diff
@@ -178,6 +178,26 @@ def find_vars(e, e_sympy):
     return variables
 
 
+def exponent(expr, var):
```

Comment:
Could you add a docstring explaining this function? It's not immediately clear what it does.